MediaAdminBundle/Tests/Functional/EventFactory/FolderEventFactoryTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace OpenOrchestra\BackofficeBundle\Tests\Functional\EventFactory;
```

Comment:
Your tests are ok on the principle but not on the form: prior to functionnal tests you should write unit tests. And all you are testing here can be tested by unit test